### DISCUSSION TITLE
string1 AND string2 should highlight string1 first

### DISCUSSION THREAD

**Discussion by mikelange49** [community/community#13040] [Other Feature Feedback, Questions, & Ideas]
Search example : burg AND juce
I'm mostly interested in files that contain "burg" (which are rare), but within files that contain "juce" (which are plenty)
That's why I put "burg" first and "juce" second.
But I mostly see "juce" highlited, not "burg"

![image](https://user-images.githubusercontent.com/34253501/159096885-9fefc910-29cf-48d4-8cac-5149b5c6f9cf.png)

**Comment by colinwm** [comment]
Good feedback. We should try to enforce some diversity in the snippets, so we capture rare terms as well as common ones. I'll look into implementing this.
